spec/integration/full_project_spec.rb

```diff
@@ -554,6 +555,33 @@
     cloned_project.delete
   end
 
+  it "should be able to export raw report" do
+    m = @project.facts.first.create_metric
+    m.identifier = 'metric.cloned_metric'
+    m.save
+
+    commits_data = [
```

Comment:
Similar code found in full_project_spec.rb:101, full_project_spec.rb:165, full_project_spec.rb:563 (mass = 66)